CMakeLists.txt

```diff
@@ -590,7 +581,5 @@ add_dependencies(shadps4 host_shaders)
 target_include_directories(shadps4 PRIVATE ${HOST_SHADERS_INCLUDE})
 
 if (ENABLE_QT_GUI)
-    set_target_properties(shadps4 PROPERTIES
-#        WIN32_EXECUTABLE ON
-        MACOSX_BUNDLE ON)
-endif()
+    set_target_properties(shadps4 PROPERTIES WIN32_EXECUTABLE ON)
```

Comment:
"WIN32_EXECUTABLE ON" this one should be disabled to have the console in Qt. You can delete the line I think

Reply:
Should I keep the MACOSX_BUNDLE ON ?